### DISCUSSION TITLE
Merge Errors

### DISCUSSION THREAD

**Discussion by vmitchell85** [spatie/flareapp.io-roadmap#51] [Ideas]
We recently had an instance where a user did the same thing multiple times but since the SQL that was failing had a different timestamp every time it created many Flare errors.

Would be great if we could merge errors together.

**Comment by AlexVanderbist** [comment]
Hi @vmitchell85, this normally shouldn't happen if the timestamp can easily be identified as a value in the SQL query. Can you send us an example over at support@flareapp.io please?
